docs/app/Examples/modules/Dropdown/Usage/DropdownExampleCloseOnBlur.js

```diff
@@ -3,7 +3,11 @@ import { Dropdown } from 'semantic-ui-react'
 import { friendOptions } from '../common'
 
 const DropdownExampleCloseOnBlur = () => (
-  <Dropdown placeholder='Select Friend' closeOnBlur fluid selection options={friendOptions} />
+  <div>
+    <Dropdown placeholder='I close on blur' closeOnBlur fluid selection options={friendOptions} />
+    <br />
```

Comment:
Let's drop the `fluid` prop and go with `{' '}` here so these are inline elements rather than full page vertically stacked elements.  Just helps keep the docs more concise.